spec/services/rss_reader_spec.rb

```diff
@@ -12,6 +12,12 @@
   let(:nonpermanent_link) { "https://medium.com/feed/@macsiri/" }
   let(:rss_data) { RSS::Parser.parse(HTTParty.get(link).body, false) }
 
+  # Override the default Rails logger as these tests require the Timber logger.
+  before do
+    timber_logger = Timber::Logger.new(nil)
+    Rails.logger = ActiveSupport::TaggedLogging.new(timber_logger)
+  end
```

Comment:
don't forget the `after`, otherwise this will stay attached to Timber in all the other spec files, as `Rails.logger` is a singleton

Reply:
AHHHH yes yes yes you're right

Reply:
@rhymes let me know if you have any ideas about that one failing test that still shows up :( i'm a little confused about that one

Reply:
@rhymes Ok, did some more investigation and figured out/fixed the issue. Turns out, the test in question was for a controller, and in the context of that controller, the logger was an entirely different class and _different from_ the `Rails.logger`. I've explicitly used `Rails.logger` in that controller to fix this, mostly because I couldn't quickly determine where the leakiness here is coming from (and it seems pretty out of scope for this PR, which started off with a completely different intention 😹).

Looking at other controllers that use logging, it seems like we have a few different ways to set messages on the logger. I propose opening an issue specifically for investigating and perhaps abstracting this out so that all controllers share one specific logger instance and so that we can consolidate around one way of logging. Thoughts?

Reply:
I'm not 100% what's the best course of action here, maybe we should consult with @mstruve, the value and content of that following PR might also depend on how much we want to be dependent on Timber in the future.

Anyway, thanks for fixing all the specs!

Reply:
Seeing all those logs I immediately want to move all of that information into Datadog but that is way outside the scope of this PR so I made a ticket for my team to do it: https://github.com/thepracticaldev/dev.to/projects/5#card-32783157

I think using `Rails.logger` is the way to go here. If we use that everywhere then switching out Timber for something else in the future is as easy as updating the production.rb file and away we go. Thanks for tackling this @vaidehijoshi